requirements/edx/base.in

```diff
@@ -31,7 +31,7 @@ boto==2.39.0                        # Deprecated version of the AWS SDK; we shou
 boto3==1.4.8                        # Amazon Web Services SDK for Python
 botocore==1.8.17                    # via boto3, s3transfer
 celery==3.1.25                      # Asynchronous task execution library
-ddt==0.8.0                          # via xblock-drag-and-drop-v2 (which probably shouldn't require it)
+ddt                                 # via xblock-drag-and-drop-v2 (which probably shouldn't require it)
```

Comment:
`via xblock-drag-and-drop-v2 (which probably shouldn't require it)` -> `Data-Driven Tests: run one test case with different data.`